pkg/services/notification/resource_control/task.go

```diff
@@ -9,58 +9,73 @@ import (
 	"time"
 
 	"openpitrix.io/logger"
-	"openpitrix.io/notification/pkg/globalcfg"
+	nfdb "openpitrix.io/notification/pkg/db"
+	"openpitrix.io/notification/pkg/global"
 	"openpitrix.io/notification/pkg/models"
 	"openpitrix.io/notification/pkg/pb"
-	"openpitrix.io/notification/pkg/util/dbutil"
+	"openpitrix.io/notification/pkg/util/pbutil"
 	"openpitrix.io/notification/pkg/util/stringutil"
 )
 
 func RegisterTask(ctx context.Context, task *models.Task) error {
-	db := globalcfg.GetInstance().GetDB()
-	tx := db.Begin()
-	err := tx.Create(&task).Error
+	db := global.GetInstance().GetDB()
+	err := db.Create(&task).Error
 	if err != nil {
-		tx.Rollback()
-		logger.Errorf(ctx, "Insert task failed, [%+v]", err)
+		logger.Errorf(ctx, "Failed to insert task, [%+v]", err)
 		return err
 	}
-	tx.Commit()
 	return nil
 }
 
-func UpdateTaskStatus(taskId, status string) error {
-	db := globalcfg.GetInstance().GetDB()
-	task := &models.Task{
-		TaskId: taskId,
-	}
-	tx := db.Begin()
-	err := db.Model(&task).Where("task_id = ?", taskId).Update("status", status).Update("status_time", time.Now()).Error
+func UpdateTasksStatus(ctx context.Context, taskIds []string, status string) ([]string, error) {
```

Comment:
[]string in ret is needed?

Reply:
_, err := rs.UpdateTasksStatus(ctx, taskIds, constants.StatusPending)
I don't use the return value when I use it. But maybe it need return value in future, so I keep it.
If you insist to remove it ,I can also remove it.

Reply:
done